containers/build.sh

```diff
@@ -64,7 +65,9 @@ while getopts htp: opt; do
             ;;
         t)  TAG=$OPTARG
             ;;
-        p)  PLATFORM=$OPTARG
+        P)  PLATFORM=$OPTARG
+            ;;
+        p)  PROFILE=$OPTARG
```

Comment:
Unfortunately there's a bit of arg collision - I'll welcome any naming suggestions to reduce confusion.

Reply:
Yeh, this is a bit tricky but I think the `-P` and `-p` the way you did it is fine.